edx-platform/pearson-theme/lms/templates/index.html

```diff
@@ -85,3 +111,56 @@ <h1 class="title">
   $(window).load(function() {$('#signup_action').trigger("click");});
 </script>
 % endif
+
+%if search_courses_home:
+  <script type="text/javascript">
+    var searchHome = (function() {
```

Comment:
@Squirrel18 it's just to structure the code as the module pattern, so first is invoked itself to create the object and then i can access its functions.

Reply:
But, you can create the object without invoked itself, since you are not using any of the functions defined, till the dom is loaded. @amalbas

Reply:
@Squirrel18 i tested without invoked itself and it doesn't work, because when i called functions after the dom is loaded, the object doesn't exist and give an error.

Reply:
Yes, because searchHome needs to be a object definition not a function definition, but if you have problems with that, let it like so. @amalbas